server/src/graphql/resolvers/content.js

```diff
@@ -0,0 +1,10 @@
+const { db } = require('../../db/models');
+
+module.exports = {
+  async content(parent) {
+    const contentArray = await db.contentText.findAll({
+      where: { type: parent.id },
+    });
+    return contentArray;
```

Comment:
you might be able to save a few lines of code by just returning the statement on line 5.